lib/prerender.js

```diff
@@ -77,13 +79,11 @@ class Prerender extends Plugin {
     pkg = JSON.parse(JSON.stringify(pkg));
     pkg.fastboot.manifest.htmlFile = this.emptyFile;
 
-
-
-
     await writeFile(path.join(this.outputPath, 'package.json'), JSON.stringify(pkg));
 
     let app = new FastBoot({
-      distPath: this.inputPaths[0]
+      distPath: this.inputPaths[0],
+      resilient: true,
```

Comment:
I think it was also intentional that we did not set `resilient` here. We want the build to fail if some of the prerendered pages can't actually prerender.